### DISCUSSION TITLE
Hey did you remove the ask AI feature ?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#1045] [Q&A]
Any problem that you have faced and think that we should also not use that ?

**Comment by LeCarteloo** [comment]
As fuma-nama said:

> This is not meant to be a feature, you need to configure it manually and choose an AI solution like inkeep or Orama Cloud.

**Reply by Gitstar-OC** [reply]
I know that I have read it, He was trying to use it in his application but Is there any reason that he is not using it now!
